cql/ManageRareAbnormality.cql

```diff
@@ -1537,6 +1609,27 @@ define TreatmentInLastYear:
 define HasTreatmentInLastYear:
   TreatmentInLastYear is not null
 
+define HasTreatmentAfterMostRecentBiopsy:
+  (MostRecentTreatmentDate is not null) and
+  MostRecentTreatmentDate after Collate.MostRecentBiopsyReport.date
+
+define PositiveSurveillanceAfterMostRecentBiopsy:
+  Coalesce(
+    (Dash.RecentPositiveHpv) R
+      where ((R.effective as FHIR.dateTime).value > Collate.MostRecentBiopsyReport.date)
```

Comment:
Consider using `Common.DiagnosticReportDate(R)` function to replace `(R.effective as FHIR.dateTime).value`. Same with 5 lines below.

Reply:
Can this be addressed in later update?

Reply:
It looks like you updated this in commit https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/120/commits/d10e412434d5ca7592a7b60ed0f7663cca282a7a. Thanks!